js/main.js

```diff
@@ -35,6 +42,18 @@ Site = {
   onResize: function() {
     var _this = this;
 
+    _this.resizeDelay();
+  },
+
+  resizeDelay: function() { 
```

Comment:
the point about code review is that the code isn't trusted until it is double checked. doesnt matter who in the team writes it none of us should trust it on spec. I wasn't sure about the scope of the timer reference variable in the debounce you have. You didnt comment to say it came from a specific source so I don't know it is existing proven code and you made it sound like you wrote it when i asked the question. using a well know debounce code snippet/jquery function as a library is KISS